docs/docs/themes/using-multiple-gatsby-themes.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Using Multiple Gatsby Themes
+título: Usando de múltiples temas de Gatsby
 ---
 
-Gatsby themes are intended to be composable. This means you can install multiple themes alongside each other.
+Gatsby temas están destinados a ser componible. Esto significa que puede instalar temas múltiples el uno junto al otro.
 
-The `gatsby-starter-theme` composes two Gatsby themes: `gatsby-theme-blog` and `gatsby-theme-notes`
+La `gatsby-starter-theme` compone dos temas de Gatsby: `gatsby-theme-blog` y `gatsby-theme-notes`
```

Comment:
```suggestion
La `gatsby-starter-theme` se compone de dos temas de Gatsby: `gatsby-theme-blog` y `gatsby-theme-notes`
```

Reply:
La `gatsby-starter-theme` => El `gatsby-starter-theme`.

Tema en castellano es masculino.

**e.g.**
[Los temas de sitios que no sean GPL...](https://es.wordpress.org/themes/getting-started/)

Reply:
Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.